climada/entity/exposures/litpop/litpop.py

```diff
@@ -233,6 +237,7 @@ def set_nightlight_intensity(self, countries=None, shape=None, res_arcsec=15,
         if countries is not None and shape is not None:
             raise ValueError("Not allowed to set both `countries` and `shape`. Aborting.")
         if countries is not None:
+            # TODO: replace with LitPop.from_countries
             self.set_countries(countries, res_arcsec=res_arcsec,
```

Comment:
In some cases, it might make sense to postpone the replacement of the `set_`-method with the new constructor to a later point. Make sure to add a `TODO` comment in those cases.

In this example, the `set_nightlight_intensity` is a `set_`-method that needs to be replaced by a constructor, only in a later work step. We decide to leave the reference to `self.set_countries` within this method for now, but will replace it as soon as we deal with the `set_nightlight_intensity` method in one of the next work steps.